.github/workflows/test.yaml

```
@@ -0,0 +1,14 @@
+name: test
+
+on: [push]
```

I think we'll want this workflow to trigger for pull requests as well, so I would replace this with something that looks like this:

```suggestion
on:
  pull_request:
    types: [opened, synchronize, reopened]
  push:
    branches:
      #Trigger workflow on push to any branch or branch heirarchy:
      - '**'
```

---

Just realized there's 'Commit suggestion' for these...  Lesson learned!

Also, great idea!  Updated!